c/shared/source/uforead/uforead.c

```diff
@@ -1228,7 +1228,16 @@ static GLIF_Rec* findGLIFRecByName(ufoCtx h, char *glyphName)
 
     GLIF_Rec* glif = bsearch(glifKey, h->data.glifRecs.array, h->data.glifRecs.cnt, sizeof(GLIF_Rec), glifRecNameComparator);
```

Comment:
I'm not sure if you're keeping this code around for future use but if we're reverting we should probably comment the bsearch and related lines out. (And I suppose if we do keep the lines we should probably keep the removed qsort in a comment as well.)

Reply:
Ah sorry about this — I pushed this in a rush earlier as I was leaving for something & was looking over my PR again just now (should've get it in draft mode until I had re-reviewed). Needs some cleaning, thanks!